scripts/shared/deploy.sh

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env bash
+set -em
+
+source ${SCRIPTS_DIR}/lib/debug_functions
+source ${SCRIPTS_DIR}/lib/version
+source ${SCRIPTS_DIR}/lib/utils
+
+### Functions ###
+
+function import_images() {
+    docker tag quay.io/submariner/submariner:$VERSION localhost:5000/submariner:local
```

Comment:
where do we set this $VERSION for the runs in shipyard?

Reply:
It's set by importing `version` script. Mind you this code is copied as is from submariner :)